<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CurlyBracesRemover</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>381</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_inputFile">
        <property name="text">
         <string>Input file path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_selectFile">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_outputFile">
      <property name="text">
       <string>Output file path</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_rmAndSave">
      <property name="text">
       <string>Remove Braces and Save!</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
